app/controllers/providers_controller.rb

```diff
@@ -7,8 +7,13 @@ class ProvidersController < ApplicationController
   before_action :build_training_provider, only: %i[training_provider_courses]
 
   def index
-    @providers = Kaminari.paginate_array(Provider.where(recruitment_cycle_year: Settings.current_cycle).all)
-                          .page(params[:page] || 1).per(params[:per_page] || 10)
+    page = (params[:page] || 1).to_i
+    per_page = 10
```

Comment:
Does this need to be hardcoded? What about a `per_page` param?

Reply:
sure i can change

Reply:
ok second thought, i will leave as is. it's hard coded on the API side so would require another change there. It's only used here so the hard code is fine i think. I don't really see this changing so will leave for now